docs/rules/color-no-hex.md

```diff
@@ -1,4 +1,4 @@
-# Prevent the use of static hex color values (`@metamask/design-tokens/color-no-hex`)
+# Prevent the use of hex color values (`@metamask/design-tokens/color-no-hex`)
```

Comment:
Aligning with description in rule https://github.com/MetaMask/eslint-plugin-design-tokens/blob/main/src/rules/color-no-hex.ts#L16

<img width="363" alt="Screenshot 2024-04-10 at 4 34 46 PM" src="https://github.com/MetaMask/eslint-plugin-design-tokens/assets/8112138/78957b0c-0712-4ee3-ba9a-e1a1e91309a4">